tests/requirements.in

```diff
@@ -4,5 +4,5 @@ coverage
 docutils
 flake8
 mock
-pytest
+pytest<4.0.0
```

Comment:
I needed to version lock pytest because I was getting the following error.

```
RemovedInPytest4Warning: Fixture "docker_services" called directly. Fixtures are not meant to be called directly, are created automatically when test functions request them as parameters. See https://docs.pytest.org/en/latest/fixture.html for more information.
```

Reply:
Okay, I unpinned this requirement.